pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [project]
 name = "xdsl-jax"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
```suggestion
```

Reply:
This is a separate change, I need to set up the dynamic version thing from xDSL, I'd much rather this were dynamic than manually update it every time.

Reply:
Sounds good :)